xml/src/main/java/com/baeldung/xmlhtml/pojo/jaxb/html/ExampleHTML.java

```diff
@@ -0,0 +1,39 @@
+package com.baeldung.xmlhtml.pojo.jaxb.html;
+
+import com.baeldung.xmlhtml.pojo.jaxb.html.elements.Body;
+import com.baeldung.xmlhtml.pojo.jaxb.html.elements.Meta;
+
+import javax.xml.bind.annotation.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@XmlAccessorType()
```

Comment:
Since the default access type is XmlAccessType.PUBLIC_MEMBER, let's either specify it, as in:
@XmlAccessorType(XmlAccessType.PUBLIC_MEMBER)
or omit the @XmlAccessorType() altogether, on all the POJOs, since it doesn't add any information.

Reply:
Oops! Removed. Thanks!